### ISSUE TITLE
Quiver XML coverage parameter

### ISSUE THREAD

**Issue by sarahbuzz** [PacificBiosciences/Bioinformatics-Training#45]
Hello,

I am attempting to run SMRTanalysis (v2.3) on the command-line using the XML protocols. However, I'd like to modify the maximum coverage parameter to be above the default of 100 in Quiver. I've attempted to add the "coverage" parameter to the XML files that call Quiver or use Quiver.1.xml but that is not resetting the parameter correctly (e.g. we are still seeing a coverage of no more than 100 in every corrections.gff file that is output from the HGAP2 pipeline, average coverage is well over 100).

Any assistance on how to modify this parameter in the XML files would be great!

Thanks,

Sarah

**Comment by rhallPB** [comment]
It's likely that the parameter not being passed is a bug. I would suggest running the quiver consensus in isolation using the `./workflow/P_AssemblyPolishing/callConsensus.sh` file and manually changing the parameter `--coverage COVERAGE.` Although using >100 will likely not make any difference to the final consensus quality. What is the reason for wanting to use more than 100x for polishing?

**Comment by 0xaf1f** [comment]
> Although using >100 will likely not make any difference to the final consensus quality. What is the reason for wanting to use more than 100x for polishing?

We've noticed worse results with 2 cells vs 1 on a bacterial genome until raising the maximum coverage. I filed a detailed issue about this in the GenomicConsensus repository, but it was closed (with your same explanation and that users can change it if they want) and the whole issues section was deleted.

**Comment by sarahbuzz** [comment]
Thank you for your help @rhallPB. @0xaf1f and I have both worked on this and have been trying to find a better solution to this problem. We need more than 100x coverage since our genomes contain much heterogeneity, which affects polishing at every step. We are aiming to analyze every read possible.

I have found a more permanent solution for our purposes. I have updated the default value of the coverage parameter in our installation of SMRTanalysis for Quiver. Within our `smrtanalysis/install` location I updated `analysis/lib/python2.7/GenomicConsensus/options.py` line 146 within the argument parsing of `"--coverage"`. I changed it to 1000. I ran the HGAP protocol with SMRTanalysis v2.3 and our `corrections.gff` file has more than 100x coverage in several positions.

I would recommend making the parameter available to change more easily since many don't even know this threshold exists, which may be producing unexpected results. This situation happened to us and produced variable results.
